.gitignore

```diff
@@ -0,0 +1 @@
+*.DS_Store
```

Comment:
- [ ] `.gitignore` would only ignore any changes made to a file, including deletion, so even if you deleted the file, it will stay on the repo if you add it to `.gitignore`, the solution is to use `git rm --cached .DS_Store` then commit and push, if you want to stop occurrences like this to happen, you can ignore the file globally, read more [here](https://code.likeagirl.io/how-to-get-rid-of-ds-store-and-node-modules-in-git-repositories-d37b8a391247)